intelmq/lib/cache.py

```diff
@@ -24,6 +24,9 @@ class Cache():
 
     def __init__(self, host: str, port: int, db: str, ttl: int,
                  password: Optional[str] = None):
+        if host is None:
+            return
```

Comment:
That is only applicable because the only code calling this cache is the bots.py for the statistics. If any other code would use this class, this fix silently ignores misconfiguration